pkg/controllers/subnet/subnet_controller.go

```diff
@@ -76,7 +78,24 @@ func (r *SubnetReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 				obj.Spec.IPv4SubnetSize = vpcNetworkConfig.DefaultIPv4SubnetSize
 			}
 		}
-		if _, err := r.Service.CreateOrUpdateSubnet(obj, nil); err != nil {
+
+		var tags []model.Tag
+		namespace := &v1.Namespace{}
+		namespacedName := types.NamespacedName{
+			Name: req.Namespace,
+		}
+		if err := r.Client.Get(context.Background(), namespacedName, namespace); err != nil {
+			log.Error(err, "unable to fetch namespace of Subnet CR", "req", req.NamespacedName)
+			updateFail(r, &ctx, obj)
+			return ResultRequeue, err
+		}
+		namespace_uid := namespace.UID
+		// user create subnet CR, it is only for VM subnet, no need to add TagScopeNamespaceUID/TagScopeNamespace
```

Comment:
no need to add TagScopeNamespaceUID/TagScopeNamespace for pod subnet? is more clear?